nix/elasticsearch.nix

```diff
@@ -0,0 +1,196 @@
+# Based on https://github.com/cachix/devenv/blob/main/src/modules/services/elasticsearch.nix
+# Note: 
+#   To use elastic search you will need the following inside your `perSystem`, if not already configured:
+#   { 
+#     _module.args.pkgs = import inputs.nixpkgs {
+#        inherit system;
+#        # Required for elastic search
+#        config.allowUnfree = true;
+#      };
+#   }
```

Comment:
This could be put in submodule's `description`?

(nix-browser can render it ultimately, so can flake.parts website)